### ISSUE TITLE
Unexpected Error when vewing Channel Messages. Can not view channel messages

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#421]
Receiving "An unexpected error has occured. Would you like to submit this error to Mirth Project.org?"

Submitted the error several times including e-mail address. The following is the content of the error message.

com.webreach.mirth.client.core.ClientException: com.webreach.mirth.client.core.ClientException: method failed: HTTP/1.1 500 Java heap space
com.webreach.mirth.client.core.MessageListHandler.getMessagesByPage(MessageListHandler.java:143)
com.webreach.mirth.client.core.MessageListHandler.getFirstPage(MessageListHandler.java:96)
com.webreach.mirth.client.ui.browsers.message.MessageBrowser.getMessageTableData(MessageBrowser.java:505)
com.webreach.mirth.client.ui.browsers.message.MessageBrowser.access$1000(MessageBrowser.java:87)
com.webreach.mirth.client.ui.browsers.message.MessageBrowser$1MessageWorker.doInBackground(MessageBrowser.java:1307)
com.webreach.mirth.client.ui.browsers.message.MessageBrowser$1MessageWorker.doInBackground(MessageBrowser.java:1293)
org.jdesktop.swingworker.SwingWorker$1.call(Unknown Source)
java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
java.util.concurrent.FutureTask.run(Unknown Source)
org.jdesktop.swingworker.SwingWorker.run(Unknown Source)
java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
java.lang.Thread.run(Unknown Source)

Imported Issue. Original Details:
Jira Issue Key: MIRTH-411
Reporter: olmeda9535
Created: 2007-07-10T08:58:34.000-0700

**Comment by rbeckman-nextgen** [comment]
Increase your heap size or decrease the number of messages per page.

Imported Comment. Original Details:
Author: brendanh
Created: 2007-07-18T11:48:05.000-0700

**Comment by rbeckman-nextgen** [comment]
Thank you looking in to this issue. I did increase the heap size. And the number of messages per page was filtered to sent messages only. I noticed that the error is only occurring when I look for transformed channel messages. It could be the type of transaction I am converting. It includes JPG encoded data. 



Imported Comment. Original Details:
Author: olmeda9535
Created: 2007-07-18T13:29:27.000-0700
